presto-main/src/main/java/io/prestosql/operator/CyclingGroupByHash.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.operator;
+
+import com.google.common.collect.ImmutableList;
+import io.prestosql.spi.Page;
+import io.prestosql.spi.PageBuilder;
+import io.prestosql.spi.block.BlockBuilder;
+import io.prestosql.spi.type.Type;
+import org.openjdk.jol.info.ClassLayout;
+
+import java.util.List;
+
+import static io.prestosql.spi.type.BigintType.BIGINT;
+
+/**
+ * GroupByHash that provides a round robin group ID assignment.
+ */
+public class CyclingGroupByHash
```

Comment:
Did you mean to put this in `main` instead of `test`?

Reply:
Well it's used for the benchmark classes which aren't in test. I assume that rules out test then?